idaes/models/properties/general_helmholtz/helmholtz_state.py

```diff
@@ -50,10 +50,119 @@
+        "flow_vol": DefaultScalingRecommendation.userInputRecommended,
+        "enth_mol": 1e-3,
+        "enth_mass": 1e-3,
+        "temperature": 1e-1,
+        "pressure": 1e-6,
+        "vapor_frac": 10,
+        "dens_mass_phase[Liq]": 1e-2,
+        "dens_mass_phase": 10,  # Vap or Mix
+        "temperature_sat": 1e-2,
+        "pressure_sat": 1e-5,
+        "enth_mol_sat_phase[Liq]": 1e-2,
+        "enth_mol_sat_phase[Vap]": 1e-4,
+        "enth_mass_sat_phase[Liq]": 1e-2,
+        "enth_mass_sat_phase[Vap]": 1e-4,
+        "dh_vap_mol": 1e-4,
+        "dh_vap_mass": 1e-4,
+        "energy_internal_mol_phase[Liq]": 1e-2,
+        "energy_internal_mol_phase": 1e-4,  # Vap or Mix
+        "energy_internal_mass_phase[Liq]": 1e-2,
+        "energy_internal_mass_phase": 1e-4,  # Vap or Mix
+        "enth_mol_phase[Liq]": 1e-2,
+        "enth_mol_phase[Vap]": 1e-4,
+        "entr_mol_phase[Liq]": 1e-1,
+        "entr_mol_phase[Vap]": 1e-1,
+        "enth_mass_phase[Liq]": 1e-2,
+        "enth_mass_phase[Vap]": 1e-4,
+        "entr_mass_phase[Liq]": 1e-1,
+        "entr_mass_phase[Vap]": 1e-1,
+        "cp_mol_phase[Liq]": 1e-2,
+        "cp_mol_phase[Vap]": 1e-2,
+        "cv_mol_phase[Liq]": 1e-2,
+        "cv_mol_phase[Vap]": 1e-2,
+        "cp_mass_phase[Liq]": 1e-3,
+        "cp_mass_phase[Vap]": 1e-3,
+        "cv_mass_phase[Liq]": 1e-3,
+        "cv_mass_phase[Vap]": 1e-3,
+        "dens_mol_phase[Liq]": 1e-2,
+        "dens_mol_phase[Vap]": 1,
+        "phase_frac": 10,
+        "energy_internal_mol": 1e-3,
+        "energy_internal_mass": 1e-3,
+        "entr_mol": 1e-1,
+        "entr_mass": 1e-1,
+        "cp_mol": 1e-2,
+        "cv_mol": 1e-2,
+        "cp_mass": 1e-2,
+        "cv_mass": 1e-2,
+        "dens_mass": 1,
+        "dens_mol": 1e-3,
+        "heat_capacity_ratio": 10,
+    }
+
+    def variable_scaling_routine(self, model, overwrite=False, submodel_scalers=None):
+        if model.amount_basis == AmountBasis.MOLE:
+            self.scale_variable_by_default(model.flow_mol, overwrite=overwrite)
+        elif model.amount_basis == AmountBasis.MASS:
+            self.scale_variable_by_default(model.flow_mass, overwrite=overwrite)
+        else:
+            raise NotImplementedError(
+                f"Scaling has been not been implemented for the AmountBasis {model.amount_basis}."
+            )
+        self.scale_variable_by_default(model.flow_vol, overwrite=overwrite)
+        if self.get_scaling_factor(model.flow_vol) is None:
+            nom_flow_mass = self.get_expression_nominal_value(model.flow_mass)
+            # Geometric mean of the mass density of steam at 100 C and 1 MPa and liquid water
+            nom_dens_mass = pyo.sqrt(1000 * 5)
+            self.set_component_scaling_factor(
+                model.flow_vol, 1 / (nom_flow_mass * nom_dens_mass), overwrite=overwrite
+            )
+
+        for varname in self.default_scaling_factors.keys():
```

Comment:
So it seems like this for loop would only set the scaling factor for flow_vol if the code in Lines 140-141 hasn't already set it. Is that correct? And do we want to give priority to the method above (i.e.  if both methods are trying to set the scaling factor, is there a reason we should defer value given by the default scaling factor?)